src/test/resources/farsandra.properties

```diff
@@ -0,0 +1,10 @@
+### Farsandra Properties
+cassandra.package.name.prefix=apache-cassandra-
+cassandra.package.name.suffix=-bin.tar.gz
+cassandra.package.dist.site=http://archive.apache.org/dist/cassandra/
+cassandra.config.file.name=cassandra.yaml
+cassandra.environment.file.name=cassandra-env.sh
+farsandra.home.folder=.farsandra
+farsandra.conf.dir=conf
```

Comment:
We want this to be able to be configured per instance for those running N copies of cassandra in the same test.